code/modules/reagents/chemistry/reagents/medicine_reagents.dm

```diff
@@ -1105,3 +1105,107 @@
 	id = "corazone"
 	description = "A medication used to treat pain, fever, and inflammation, along with heart attacks."
 	color = "#F5F5F5"
+
+/datum/reagent/medicine/ketrazine
+	name = "Ketrazine"
+	id = "ketrazine"
+	description = "A powerful and addictive combat stimulant, capable of healing grievous wounds and enabling the user to shrug off stuns and heavy weights by stimulating tendons and muscle groups; however the strain on the body causes severe lasting damage. Use only in life-or-death situations. Overdose is almost invariably fatal."
+	reagent_state = LIQUID
+	color = "#5F42F4"
+	metabolization_rate = 0.5 * REAGENTS_METABOLISM
+	overdose_threshold = 20
+	addiction_threshold = 5
+
+/datum/reagent/medicine/ketrazine/on_mob_life(mob/living/M)
+	M.status_flags |= IGNORESLOWDOWN
+	M.status_flags |= GOTTAGOFAST
+	M.adjustToxLoss(-3*REM, 0)
+	M.adjustBruteLoss(-5*REM, 0)
+	M.adjustFireLoss(-5*REM, 0)
+	M.adjustOxyLoss(-5*REM, 0)
+	M.adjustStun(-80*REM, 0)
+	M.adjustKnockdown(-70*REM, 0)
+	M.adjustStaminaLoss(-80*REM, 0)
+	M.adjustUnconscious(-50*REM, 0)
+	switch(current_cycle)
+		if(13)
+			to_chat(M, "<span class='warning'>You feel incredibly powerful! Nothing can stop you!</span>")
+		if(13 to 15)
+			to_chat(M, "<span class='warning'>Your muscles begin to ache terribly...</span>" )
+		if(15 to 25)
+			to_chat(M, "<span class='warning'>You feel like your body is being ripped to shreds!</span>")
+			M.drowsyness += 3
+			M.adjustBruteLoss(10*REM, 0)
+			M.adjustToxLoss(7*REM, 0)
+		if(25 to 30)
+			to_chat(M, "<span class='warning'>The pain is unbearable! You can barely stand!</span>")
+			M.Sleeping(40, 0)
+			M.adjustKnockdown(40*REM,0)
+			M.drop_all_held_items()
+			M.Dizzy(3)
+			M.drowsyness +=4
+			M.adjustBruteLoss(15*REM,0)
+			M.adjustToxLoss(10*REM,0)
+			M.adjustStaminaLoss(30*REM,0)
+		if(30 to INFINITY)
+			to_chat(M. "<span class='warning'>Your body	can't handle the stress!</span>")
+			M.Sleeping(60, 0)
+			M.adjustKnockdown(70*REM,0)
+			M.drop_all_held_items()
+			M.Dizzy(5)
+			M.drowsyness +=6
+			M.adjustBruteLoss(20*REM,0)
+			M.adjustToxLoss(15*REM,0)
+			M.adjustStaminaLoss(40*REM,0)
+			M.losebreath +=1
+
+			=1
```

Comment:
=1?

Reply:
Is that not required at the end of a (switch)? I was under the impression that it was.